### PR TITLE
GameTeam Factory/GameTeam Class

### PR DESCRIPTION
**Pull request by zach-bergman**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [X] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change
Created GameTeamFactory and GameTeam classes, as well as tests for both. Followed TDD.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before sucmitting PR
- [X] I assigned atleast 2 group members to review my PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ignore the simplecov coverage directory
```
Comment (noahdurbin) [inline .gitignore:1]: thanks for adding this, I didn't know that we could tell git to ignore files before this

File: spec/game_team_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        shots: 8,
+        tackles: 44,
+        pim: 8,
+        ppo: 3,
+        ppg: 0,
+        faceoff_win_per: 44.8,
+        giveaways: 17,
+        takeaways: 7
+    }
+    @gameteam_attr_test = GameTeam.new(gameteam_data)
```
Comment (noahdurbin) [inline spec/game_team_spec.rb:22]: great naming, I like that it is really clear and know exactly what it's for in this line

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -6,7 +6,9 @@
     config.formatter = :documentation
 end
 
-require 'pry'
```
Comment (noahdurbin) [inline spec/spec_helper.rb:9]: there was a merge conflict here that I was able to resolve, nothing major, but was able to just rearrange and make it a little more organized at the same time.

File: spec/game_team_spec.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+end
+
+describe "initialize" do
+    it "exists" do
+        expect(@gameteam_attr_test).to be_an_instance_of(GameTeam)
+    end
+
+    it "has readable attributes" do
```
Comment (noahdurbin) [inline spec/game_team_spec.rb:30]: I like these tests too, in my class init I only tested for the data type so far but I think it's good to add testing to make sure we're getting the exact data we want as well like you did here

**Review by noahdurbin — APPROVED** [review]
looks great, we are just about ready to start making some stat trackers!

File: lib/game_team_factory.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+require "csv"
+
+class GameTeamFactory
+  @@game_teams = []
```
Comment (austincarrjones) [inline lib/game_team_factory.rb:4]: I didn't end up using a class variable on mine, but will look forward to revisiting that lesson and possibly refactoring mine to replicate this.

File: spec/game_team_spec.rb (hunk 2 above)
Comment (austincarrjones) [reply spec/game_team_spec.rb:22]: ah nice, I was confused at first "what's a attr_test? is that like a attr_accessor that I've never used?" then realized it's just a test variable. Makes sense!

**Review by austincarrjones — APPROVED** [review]
Great start!

**Pull request merged**